python/cudf/cudf/core/scalar.py

```diff
@@ -471,6 +458,9 @@ class Scalar(BinaryOperand, metaclass=CachedScalarInstanceMeta):
     _VALID_BINARY_OPERATIONS = BinaryOperand._SUPPORTED_BINARY_OPERATIONS
 
     def __init__(self, value, dtype=None):
+        warnings.warn(
```

Comment:
Some tests are failing because of this warning. I guess there are still some places where we use `cudf.Scalar`?

Reply:
Yup, thanks. I haven't merged in https://github.com/rapidsai/cudf/pull/18331 yet from branch 25.06 which _should_ be the last internal usage of `cudf.Scalar`